packages/app/src/cli/services/dev/processes/setup-dev-processes.test.ts

```diff
@@ -107,6 +108,8 @@ describe('setup-dev-processes', () => {
 
     const graphiqlKey = 'somekey'
 
+    process.env.SHOPIFY_CLI_ENABLE_APP_LOG_POLLING = '1'
```

Comment:
Should we add a test to ensure that when this flag is 0 the setup is not performed?

Reply:
Sure thign! I can just split out our change to the dev process list into its own test.
Then when we release and remove the env flag, we just delete our test and move the check back into the main one.